frontend/src/scenes/settings/organization/Members.tsx

```diff
@@ -258,6 +258,21 @@ export function Members(): JSX.Element | null {
             },
             sorter: (a, b) => a.joined_at.localeCompare(b.joined_at),
         },
+        {
+            title: 'Last Signed In',
```

Comment:
Nit: We write things using "Sentence case" at PostHog, and we prefer "logging in" over "signing in"
```suggestion
            title: 'Last logged in',
```